bundles/org.eclipse.swt/Eclipse SWT/common/org/eclipse/swt/layout/TabFolderLayout.java

```diff
@@ -0,0 +1,68 @@
+/*******************************************************************************
+ * Copyright (c) 2000, 2005 IBM Corporation and others.
+ *
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+
+package org.eclipse.swt.layout;
+
+import org.eclipse.swt.*;
+import org.eclipse.swt.graphics.*;
+import org.eclipse.swt.widgets.*;
+
+/**
+ * This layout controls the position and size
+ * of the children of a tab folder.
+ *
+ * @since 3.127
+ */
+class TabFolderLayout extends Layout {
```

Comment:
Can this really be package private?

Reply:
@deepika-u You haven't answered here? How is this supposed to be used from platform.ui?

Reply:
Yeah totally agree when no modifier is mentioned it has defaulted to private. For this class to be available for platform.ui it needs to be public. It is done now.

Reply:
Is anything being missing to be done from my side? Please do let me know.